03/index.html

```diff
@@ -4,144 +4,201 @@
+        Because you lighten up my heart when I start to cry
+      </p>
     </article>
     <section>
-      <h5>
-        He llorado tantos días, tantas noches
-        Haciendo el duelo, el duelo por ti
-        Lo dijiste claramente no querías
-        No podías imaginarte un futuro junto a mí
-        Pero cada tanto vuelves, me llamas
-        Y sabes que jamás voy a decir que no
-        Si, cada tanto vuelves
-        Basta, te pido por favor
-      </h5>
-    </section>
-    <section>
-      <h4>Chorus</h4>
-      <h5>
-        Vete de una vez y ya no vuelvas
-        Que mi corazón está cansado y ya no puede más
-        Ya perdí todo el honor, llegas y matas lo poco
-        De cordura que me queda y no me dejas avanzar
-        Vete de una vez, que me haces daño
-        ¿No ves que mi alma tiene un límite para el dolor?
-        Yo te di todo mi amor, no lo quisiste y ahora
-        Debes aprender a estar sin mí
-      </h5>
-    </section>
-    <footer>
-      <p>Fuente: Musixmatch</p>
-      <p>Compositores: Maria Daniela Spalla</p>
-      <p> Letra de Vete de una vez © Sei Track Music S.a. De C.v.</p>
-    </footer>
-    <header>lovely</header>
-    <nav>
-      <b>Billie Eilish feat Khalid</b>
-    </nav>
-    <section>
-      <h3> 
-        <i>Intro</i>
-      </h3>
-    </section>
-    <section>
+      <h3>Chorus</h3>
       <p>
-        Thought I found a way
-        Thought I found a way out (found)
-        But you never go away (never go away)
-        So I guess I gotta stay now
-        Oh, I hope some day I'll make it out of here
-        Even if it takes all night or a hundred years
-        Need a place to hide, but I can't find one near
-        Wanna feel alive, outside I can't fight my fear
+        Forgive me first love, but I'm tired
+        I need to get away to feel again
+        Try to understand why
+        Don't get so close to change my mind
+        Please wipe that look out of your eyes
+        It's bribing me to doubt myself
+        Simply, it's tiring
       </p>
     </section>
-    <aside>
-      <h3>
-        <i>Coro </i>
-      </h3>
-    </aside>
+  </main>
+  <div>
+    <h3>Outro</h3>
+    <p>
+      This love has dried up and stayed behind
+      And if I stay I'll be alive
+      Then choke on words I'd always hide
+      Excuse me first love, but we're through
+      I need to taste a kiss from someone new
+      Forgive me first love, but I'm too tired
+      I'm bored to say the least and I, I lack desire
+      Forgive me first love
+      Forgive me first love
+      Forgive me first love
+      Forgive me first love
+      Forgive me
+      Forgive me first love
+      Forgive me first love
+    </p>
+  </div>
+  <footer>
+    <p>
+      <i>Fuente: Musixmatch</i>
+    </p>
+    <p>Compositores: 
+      <strong>Laurie Blue Adkins</strong>
+    </p>
+    <p>
+      <mark>Letra de First Love © Melted Stone Publishing Ltd.</mark>
+    </p>
+  </footer>
+  <header>Vete de una vez</header>
```

Comment:
The same as the last requested change
Is good to have plain text inside a container tag?